docs/octant-plugin-installation.md

```diff
@@ -0,0 +1,107 @@
+# octant and antrea-octant-plugin installation
+
+## Overview
+
+There are two ways to deploy octant and antrea-octant-plugin.
+
+* Deploy octant and antrea-octant-plugin as a process (out of Kubernetes cluster).
+
+* Deploy octant and antrea-octant-plugin as a pod (inside Kubernetes cluster).
+
+### Before you start
+antrea-octant-plugin depends on monitoring CRDs, AntreaControllerInfo and AntreaAgentInfo.
+
+To run octant together with antrea-octant-plugin, please make sure you have these two CRDs defined in you K8s cluster.
```

Comment:
If monitoring module runs fine, there will be CRD added. And there will be no problem for UI even if there are no members created, then we will show empty tables with only headers of each column.

But the problem I want to raise here is that the definitions of AntreaAgent/ControllerInfo must exist in the cluster, otherwise antrea-octant-plugin and in fact, octant navigation bar itself will encounter problems just like it has appeared in our previous set up. 

I have thought of copying the monitoring CRDs definitions to antrea-octant.yml, but I am afraid that in that case if someone accidentally deletes antrea-octant.yml, then antrea monitoring module will encounter problem. 

For the current yaml files, we need to create antrea.yml first, then antrea-octant.yml. For deletion, delete antrea-octant.yml and then antrea.yml. Maybe I should also add this information to the document? @jianjuns

Reply:
My question is - if the CRD definition is not added, will Antrea controller and agent be able to run? If not, we need not to say CRD is a must for Octant plugin separately; or at least we should say "if Antrea is not deployed, then you need to at least add CRDs".

Could we fix the Octant plugin issues when CRD definition is missing?

Reply:
If the CRD definition is not added, controller and agent are able to run since monitor is a separate goroutine. But monitoring goroutine will exit in this case because creatinon operations will get error.

If the CRD definition is missing, I think we need to query all the existing CRD types to fix the octant plugin issue. Let me take a further look at this part.

Reply:
Got it. Here at least say if Antrea is deployed before Octant plugin starts using the standard deployment yaml, CRDs should already be added.